autoPyTorch/api/base_task.py

```diff
@@ -122,6 +122,241 @@ def get_search_updates():
+        value_range=['shake-shake'],
+        default_value='shake-shake',
+    )
+    # mlp backbone
+    search_space_updates.append(
+        node_name='network_backbone',
+        hyperparameter='ShapedMLPBackbone:mlp_shape',
+        value_range=['funnel'],
+        default_value='funnel',
+    )
+    search_space_updates.append(
+        node_name='network_backbone',
+        hyperparameter='ShapedMLPBackbone:num_groups',
+        value_range=[1, 5],
+        default_value=2,
+    )
+    search_space_updates.append(
+        node_name='network_backbone',
+        hyperparameter='ShapedMLPBackbone:output_dim',
+        value_range=[64, 1024],
+        default_value=64,
+        log=True
+    )
+    search_space_updates.append(
+        node_name='network_backbone',
+        hyperparameter='ShapedMLPBackbone:max_units',
+        value_range=[64, 1024],
+        default_value=64,
+        log=True
+    )
+    search_space_updates.append(
+        node_name='network_backbone',
+        hyperparameter='ShapedMLPBackbone:activation',
+        value_range=['relu'],
+        default_value='relu',
+    )
+
+    # training updates
+    # lr scheduler
+    search_space_updates.append(
+        node_name='lr_scheduler',
+        hyperparameter='__choice__',
+        value_range=['CosineAnnealingLR'],
+        default_value='CosineAnnealingLR',
+    )
+    search_space_updates.append(
+        node_name='lr_scheduler',
+        hyperparameter='CosineAnnealingLR:T_max',
+        value_range=[50],
+        default_value=50,
+    )
+    # optimizer
+    search_space_updates.append(
+        node_name='optimizer',
+        hyperparameter='__choice__',
+        value_range=['AdamOptimizer', 'SGDOptimizer'],
+        default_value='AdamOptimizer',
+    )
+    # adam
+    search_space_updates.append(
+        node_name='optimizer',
+        hyperparameter='AdamOptimizer:lr',
+        value_range=[1e-4, 1e-1],
+        default_value=1e-3,
+        log=True
+    )
+    # sgd
+    search_space_updates.append(
+        node_name='optimizer',
+        hyperparameter='SGDOptimizer:lr',
+        value_range=[1e-4, 1e-1],
+        default_value=1e-3,
+        log=True
+    )
+    search_space_updates.append(
+        node_name='optimizer',
+        hyperparameter='SGDOptimizer:weight_decay',
+        value_range=[1e-5, 1e-1],
+        default_value=1e-3,
+    )
+    search_space_updates.append(
+        node_name='optimizer',
+        hyperparameter='SGDOptimizer:momentum',
+        value_range=[0.1, 0.999],
+        default_value=0.1,
+        log=True
+    )
+    search_space_updates.append(
+        node_name='data_loader',
+        hyperparameter='batch_size',
+        value_range=[16, 512],
+        default_value=128,
+        log=True
+    )
+
+    # preprocessing
+    search_space_updates.append(
+        node_name='feature_preprocessor',
+        hyperparameter='TruncatedSVD:target_dim',
+        value_range=[0.1, 0.9],
```

Comment:
Did this change to percentage-wise from the old implementation ?

Reply:
yes